deptrac.yaml

```diff
@@ -7,6 +10,8 @@ deptrac:
   paths:
     - ./src
 
+  cache_file: my.deptrac.cache
```

Comment:
Minor thing, but as far as I can tell, the cache keys usually use camelCase, so `cacheFile` might be more consistent

Reply:
Really? `skip_violations`, `exclude_files`. We use camelCase via the php-config and `snake_case` via `yaml`.